src/settings.yml.sample

```diff
@@ -117,3 +117,76 @@ pinanas:
 
       # Send emails as. Type: string. Example: "contact@{{ pinanas.domain }}".
       sender:
+
+
+  applications:
+    #
+    # === SELECT YOUR APPS === #
+    #
+    # PiNANAS includes following apps by default:
+    # - DHCPD
+    # - Traefik
+    # - Fail2ban
+    # - Authelia
+    #
+    # Optional applications can also be activated according to your needs. For a brief description of theses apps, go read the chapter "APPS DESCRIPTION" at the end of this file.
```

Comment:
I'd rather have optional applications enabled by default to avoid confusion. PiNanas should work out-of-the-box, and those applications are listed on the README.md. Additional work is necessary to better configure PiNanas to one's need, and this is expected. But without any additional work, all items listed on the box should be in the box.